stable/cosi-bucket-kit/templates/bucketclasses.yaml

```diff
@@ -0,0 +1,14 @@
+{{- range .Values.bucketClasses }}
+apiVersion: objectstorage.k8s.io/v1alpha1
+kind: BucketClass
+metadata:
+  name: {{ .name }}
+driverName: {{ .driverName }}
+deletionPolicy: {{ .deletionPolicy }}
+{{- if .parameters }}
+parameters:
+  {{- range $key, $value := .parameters }}
+  {{ $key }}: {{ $value }}
+  {{- end }}
```

Comment:
Could we use `toYaml` here similarly to the `ceph-provider.yaml`?

Reply:
pushed [30754a9](https://github.com/mesosphere/charts/pull/1534/commits/30754a95921e1e7ee0ca5b8673572605b33d623c)